components/webui/client/src/pages/SearchPage/SearchControls/Presto/SqlSearchButton/RunButton/index.tsx

```
@@ -3,9 +3,14 @@ import {useCallback} from "react";
 import {CaretRightOutlined} from "@ant-design/icons";
 import {
     Button,
+    notification,
     Tooltip,
 } from "antd";
 
+import {
+    SyntaxError,
```

Biome (2.1.2)

[error] Do not shadow the global "SyntaxError" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)